### PR TITLE
fix(vmware): fix a cta is not in blue color

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-81603 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I fixed a cta is not in blue color.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html [hunk 1]
```diff
@@ -170,7 +170,7 @@
                 class="oui-dropdown-option mb-1"
                 role="menuitem"
                 data-ng-if="$ctrl.bindings.numberOfIPBlocks.total"
-                data-ng-click="$ctrl.onAssociateIpBlock()"
+                data-ng-href="{{:: $ctrl.onAssociateIpBlock() }}"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html:173]: if duable, it's better to rename the variable to:
```suggestion
                data-ng-href="{{:: $ctrl.associateIpBlockLink }}"
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
 
       onAssociateIpBlock: /* @ngInject */ ($state) => () =>
-        $state.go('app.dedicatedCloud.details.dashboard.associate-ip-bloc'),
+        $state.href('app.dedicatedCloud.details.dashboard.associate-ip-bloc'),
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js:54]: related to this :
https://github.com/ovh/manager/pull/6890/files#r848529281

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html]: I renamed this fonction.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js]: I renamed this function.

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by frenautvh — APPROVED** [review]

**Review by marie-j — COMMENTED** [review]
Ok with the change but I'm having trouble understanding the link with the commit message 
Maybe something like `change link` would be better

**Comment by ghyenne** [comment]
> Ok with the change but I'm having trouble understanding the link with the commit message Maybe something like `change link` would be better

Before fix the color of link it's not good.

Capture d’écran du 2022-04-15 10-25-10

**Comment by marie-j** [comment]
> > Ok with the change but I'm having trouble understanding the link with the commit message Maybe something like `change link` would be better
> 
> Before fix the color of link it's not good.
> 
> Capture d’écran du 2022-04-15 10-25-10

ok so maybe we could go with something like `uniformize style` or `use primary theme`

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**